server/server.py

```diff
@@ -111,3 +117,14 @@ def handle_team_join(event):
     """Handle instances when user joins the Launchpad slack workspace."""
     logging.info("Handled 'team_join' event")
     core.handle_team_join(event)
+
+
+def send_webhook_notif(message):
```

Comment:
I was thinking maybe this method could be put in the `Bot` class seeming as it would be an appropriate layer of abstraction. Not super critical I guess but just a thought.

Reply:
y a a , this'd mean I don't use the private __bot variable too

Reply:
Agree here too - that would also resolve the other comment.

Reply:
yep i can move `send_webhook_notif` into the `Bot` class so i dont have to mess with 'private' variables, sure. however, since team webhook notifs only have direct access to the core, i need to have a weird middle-man function in core to call the newly moved `Bot.send_webhook_notif`. is there a better way to do it?